metrics/cli/docs/06_update_command_analysis.md

```diff
@@ -0,0 +1,136 @@
+# Update feature Analysis
+> Feature description / User story.
+
+As a user, I want to have the opportunity to update an existing project, so I don't have to create a new one every time.
+
+## Contents
+
+- [**Analysis**](#analysis)
+    - [Feasibility study](#feasibility-study)
+    - [Requirements](#requirements)
+    - [Landscape](#landscape)
+    - [Prototyping](#prototyping)
+      - [Update Config](#update-config)
+      - [Updater](#updater)
+    - [System modeling](#system-modeling)
+
+# Analysis
+> Describe a general analysis approach.
+
+During the analysis stage, we are going to investigate the implementation approaches for updating existing projects with the Metrics CLI.
+
+### Feasibility study
+> A preliminary study of the feasibility of implementing this feature.
+
+For now, the Metrics CLI does not have the command to redeploy the Metrics WEB using the existing GCloud project but only deploys to the new one every time.
+There are two problems associated with the above:
+- the GCloud has account restrictions on the number of projects, and deleting an existing one takes a long time(30 days);
+- the end-user requires to configure the newly created GCloud project every Metrics Web deployment
+
+Therefore, to improve the UX of the Metrics CLI, we should implement a command which provides the ability to redeploy the Metrics WEB using the existing GCloud project.
+Since the update command is a modified deploy command, where the YAML config is used instead of the user interaction, it is possible to implement the update feature.
+
+### Requirements
+> Define requirements and make sure that they are complete.
+
+The update command implementation approach must satisfy the following requirements:
+- Designed YAML configuration structure required for the update command in the Metric CLI
+- Possibility to update an existing project using the YAML update command configuration.
+
+### Landscape
+> Look for existing solutions in the area.
+
+Deploying the metrics web to the existing GCloud project is a complex process with the custom steps, such as Sentry configuring, deploying the Metrics WEB to the hosting, etc., so the custom solution is used.
+
+Let's review the top-level approach of the desired feature:
+
+After analyzing the whole process of the update command, we have highlighted the following methods required to implement the command:
+- parse the configuration YAML
+- checkout project from the repository
+- install npm dependencies
+- build flutter web 
+- configuring Sentry
+- deploy rules and functions
+- deploy to the hosting.
+
+All methods above have already been implemented for the Deploy command, except the configuration YAML parsing.
+
+For work with the YAML config, the following is required:
+- create a model of the configuration representation
+- integrate the YAML parser into the Metrics CLI.
+
+### Prototyping
```

Comment:
I believe the `Prototyping` section should contain only the code snippets that prove that the feature implementation is possible. For example, the redeploy feature requires the selection of the GCloud project, so we should provide a code snippet that will show that we can choose the project we want to deploy the app to.